pkg/trie/triedb/constants.go

```diff
@@ -0,0 +1,7 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package triedb
+
+const FirstPrefix = 0b00 << 6
```

Comment:
What is `FirstPrefix`?

Reply:
I took it from parity's [trie](https://github.com/paritytech/trie/blob/c3348e4b335443c17586a8b7491ba893cfde76a6/test-support/reference-trie/src/substrate_like.rs#L46-L55) crate but now that I think we can simplify that logic. Let me do some changes 😄